### ISSUE TITLE
Error with my environment Laravel homestead, hhvm ... Syntax error

### ISSUE THREAD

**Issue by tonimain** [wp-sync-db/wp-sync-db#40]
When i'm using pull in my local environment i always got this error:
``` 
You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-' at line 1 f\xc3\xbcr Abfrage - von do_action('wp_ajax_wpsdb_migrate_table'), WPSDB->ajax_migrate_table, WPSDB->process_chunk
```
When i'm testing with mamp and do a local pull evrything is fine.
Do you have some ideas or known issues with this setup?

**Comment by slang800** [comment]
Laravel with WordPress? that's interesting... Anyway, I've never tested this with HHVM, but if it's easy to add compatibility then I'd be happy to take a PR for it.

**Comment by gerardreches** [comment]
@slang800 No, he was talking about [Laravel homestead](https://laravel.com/docs/5.6/homestead), not Laravel, a VM with all the necessary to run Laravel projects and that also has the necessary to run WordPress projects.
